AISKU/src/AISku.ts

```diff
@@ -52,7 +54,12 @@ const defaultConfigValues: IConfigDefaults<IConfiguration> = {
     connectionString: UNDEFINED_VALUE,
     endpointUrl: UNDEFINED_VALUE,
     instrumentationKey: UNDEFINED_VALUE,
-    diagnosticLogInterval: cfgDfValidate(_chkDiagLevel, 10000)
+    diagnosticLogInterval: cfgDfValidate(_chkDiagLevel, 10000),
+    featureOptIn:{
+        ["iKeyUsage"]: {mode: FeatureOptInMode.disable},
+        ["CdnUsage"]: {mode: FeatureOptInMode.disable},
+        ["SdkLoaderVer"]: {mode: FeatureOptInMode.disable}
```

Comment:
extract to const at top of file like

`const SDK_LOADER_VER = "SdkLoaderVer";`